docs/_blog/2019-11-27-kuma-0-3-0.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Kuma 0.3 Released with Traffic Routing!
+description: Kuma 0.3 Released with Traffic Routing!
+date: 2019-11-27
+tags:
+  - Release
+---
+
+Today, we’re thrilled to release Kuma 0.3, our open source control plane with brand new traffic routing capabilities. Kuma’s new L4 traffic routing rules allow you to easily implement blue/green deployments and canary releases. In summary, Kuma will now alleviate the stress of deploying new versions and/or features into your service mesh. Let’s take a glimpse at how to achieve it in our sample application:
+
+<center>
+<img src="/images/blog/0-3-0-blog-pic-1.jpg" alt="" style=" padding-top: 20px; padding-bottom: 10px;"/>
```

Comment:
@bloqhead wanted to check with you first, this will work in the blog section too right? When I compiled it locally, the blog section didn't work so wasn't sure how to check.

Reply:
Hey there. Let me take a look today, test it out, and I'll let you know. You should be able to have images with no issues but I want to run it myself to see what you're encountering.

Reply:
Any updates on this @bloqhead

Reply:
@devadvocado yes, you can add images in the blog section. They don't appear in the excerpt but they should work in the detail view. Just make sure to have your path be absolute like the above: `src="/images/blog/[...]"`

Can you elaborate on what isn't working? I've got the blog running locally and it's working properly for me. You may have to run `yarn install` / `npm install` to make sure the blog plugin is there (`@vuepress/plugin-blog`).

Reply:
Another thing to note: if you want your blog posts to have a masthead image, you can have it set in the `frontmatter` at the top of your blog markdown file like this:

```md
---
headerImage: /images/blog/my-header.png
---
```

There are 2 example header images located in `.vuepress/public/images/blog` (`test-header.jpg` and `test-header-2.jpg`).

- [Screenshot of the excerpt view w/ a masthead](https://share.getcloudapp.com/p9uw6159)
- [Screenshot of the detail view w/ a masthead](https://share.getcloudapp.com/OAuJ65bO)